webapps/frontend/app/viewmodels/wizards/editmaintenance/confirm.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2016 iNuron NV
```

Comment:
2018

Reply:
Why don't you spin up some code to adjust all the headers within the repo

Reply:
For new files the (C) notice the year is when the file is created, not when last updated!

Reply:
vi crontab
0 0 1 1 *: find ~repo/framework | xargs sed -i s/$(($(date +%Y) - 1))/$(date +%Y)/; git commit -am "copyright"; git push